### ISSUE TITLE
Using 'scipopts' throws an error

### ISSUE THREAD

**Issue by RobinKlement** [scipopt/MatlabSCIPInterface#4]
Unfortunately, when specifying 'scipopts' inside 'solverOpts', `scip` throws an error. Thus I cannot use 'scipots' at all.

As an example I took "MixedInteger_LinearProgramming.m" and I ran the first example with changing the code
```
opts = optiset('display','off');
```
for
```
sopts = scipset('scipopts',{'limits/solutions',1});
opts = optiset('display','off','solverOpts',sopts);
```
and I got the following error:
```
Error using scip
SCIP Options (solverOpts) should be a cell array of the form {'name1', val1; 'name2', val2}.

Error in opti_scip (line 89)
[x,fval,exitflag,stats] = scip(H, f, A, rl, ru, lb, ub, xint, sos, qc, [], x0, opts);

Error in opti/solveOpti>solveMILP (line 110)
        [x,fval,exitflag,info] = opti_scip([],p.f,p.A,p.rl,p.ru,p.lb,p.ub,p.int.str,p.sos,[],x0,opts);

Error in opti/solveOpti (line 56)
        [x,fval,exitflag,info] = solveMILP(prob,nl,x0,opts);

Error in opti/solve (line 119)
            [x,fval,exitflag,info] = solveOpti(optObj,x0,xval);

Error in MixedInteger_LinearProgramming (line 33)
[x,fval,exitflag,info] = solve(Opt,x0)
```

**Comment by RobinKlement** [comment]
[opti_debug.txt](https://github.com/scipopt/MatlabSCIPInterface/files/11090815/opti_debug.txt)

**Comment by pfetsch** [comment]
I will have a look.

**Comment by pfetsch** [comment]
Please try:
```
sopts = {'limits/solutions',1};
opts = optiset('display','off','solverOpts',sopts);
```
If this does not work, then try:
```
sopts = {'limits/solutions',1};
opts = optiset('display','off');
opts.solverOps = sopts;
```

**Comment by pfetsch** [comment]
`scipset`  should currently not be used. I tried to update the documentation to make this clear.

**Comment by RobinKlement** [comment]
Thank you for your support. I have updated MATLABScipInterface on my machine and recompiled the mex file. When trying the first variant, I get:
```
Error using optiset
Parameter 'solverOpts' should be a structure
```
With the second variant I can start SCIP but the new mex file seems to be broken and Matlab freezes, leaving me with the only option of force-quitting Matlab. The same happens with the post-installation tests. Therefore I started it with the old mex file and the optimization completed. But I have not checked if changing these parameters influences the optimization result.

[opti_debug.txt](https://github.com/scipopt/MatlabSCIPInterface/files/11286123/opti_debug.txt)
